subsys/bluetooth/host/classic/avdtp.c

```diff
@@ -343,26 +385,42 @@ static void avdtp_process_configuration(struct bt_avdtp *session,
 		uint8_t error_code = 0;
 
 		sep = avdtp_get_sep(net_buf_pull_u8(buf) >> 2);
```

Comment:
To prevent malicious requests, it is best to determine the remaining data length before pulling the buffer.

Reply:
Updated.